bitnami/thanos/Chart.yaml

```
@@ -35,4 +35,4 @@ maintainers:
 name: thanos
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/thanos
-version: 12.21.2
+version: 12.21.3
```

We should bump the minor version per semver policy as we are introducing a new feature.

```suggestion
version: 12.22.0
```

---

Yeah that's true, I was about that it is part of almost all Helm Charts, and it is missing here, so I bumped the minor. But I leave the decision up to you.